rpcserverhandlers_test.go

```diff
@@ -3488,6 +3488,80 @@ func TestHandleGetTxOutSetInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleLiveTickets(t *testing.T) {
+	t.Parallel()
+
+	tkt1, err := chainhash.NewHashFromStr("1f6631957b4060d81ba7e760ec9c8150ba028eb051ddadf2b9749a5ccda1a955")
+	if err != nil {
+		t.Fatalf("[NewHashFromStr] unexpected error: %v", err)
+	}
+	tkt2, err := chainhash.NewHashFromStr("eca7e802590df60f7d300b6170f63dfab213b26421ed2e70de3ec2224cb9e460")
+	if err != nil {
+		t.Fatalf("[NewHashFromStr] unexpected error: %v", err)
+	}
```

Comment:
You can the `mustParseHash` helper function that is defined in this file (and in the next test below as well)